src/main/java/utils/configurations/Configurations.java

```diff
@@ -0,0 +1,45 @@
+package utils.configurations;
+
+import onlineservices.Service;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import services.WindowControl.WindowControlService;
+import utils.filereaders.CsvFileReader;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class Configurations {
+
+    @Bean
+    public WindowControlService windowControlService() {
+        return new WindowControlService();
+    }
+
+    private static final Logger logger = LogManager.getLogger(Configurations.class.getName());
```

Comment:
this to the top, also did you check the logs? is the activated service in there?

Reply:
also all static constants should be UPPER_CASE